src/app/component/CookiePopup/CookiePopup.style.scss

```diff
@@ -0,0 +1,45 @@
+:root {
+	--cookiepopup-bg: #{$white};
+}
+
+.CookiePopup {
+	display: flex;
+	width: 100vw;
+	position: fixed;
+	left: 0;
+	bottom: 0;
+	align-items: center;
+	justify-content: space-between;
+	padding: 40px 60px;
+	background-color: var(--cookiepopup-bg);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649838)